src/preload/preload.ts

```
@@ -6,6 +6,14 @@
 export interface ElectronAPI {
   mulmoHandler: (method: string, ...args: unknown[]) => Promise<unknown>;
   onProgress: (callback: (...args: unknown[]) => void) => void;
+  dialog: {
+    openFile: () => Promise<string | null>;
+  };
+  file: {
+    readBinary: (
+      filePath: string,
+    ) => Promise<{ name: string; size: number; type: string; buffer: ArrayBuffer } | null>;
```

ESLint

[error] Replace `⏎······filePath:·string,⏎····` with `filePath:·string`

(prettier/prettier)